buildenv/jenkins/openjdk_tests

```diff
@@ -170,30 +170,31 @@ timestamps{
                 LABEL = params.LABEL
             } else {
                 LABEL = PLATFORM_MAP[params.PLATFORM]["LABEL"]
-                    if (params.BUILD_LIST.contains("perf")) {
-                        def perfLabel = LABEL.minus("ci.role.test&&").concat("&&ci.role.perf")
-                        if (areNodesWithLabelOnline(perfLabel)) {
+                if (params.BUILD_LIST.contains("perf")) {
+                    def perfLabel = LABEL.minus("ci.role.test&&").concat("&&ci.role.perf")
+                    if (areNodesWithLabelOnline(perfLabel)) {
                         LABEL = perfLabel
-                        }
                     }
                 }
+                if (params.TEST_FLAG == "FIPS") {
+                    LABEL = LABEL.minus("ci.role.test&&").concat("&&ci.role.test.fips")
+                }
```

Comment:
This is needed regardless `params.LABEL` is provided by the users or not.

Reply:
I'm not sure why? Same as other parameters. User should figure out the correct machine, which has the label `ci.role.test.fips` and hasn't the label `ci.role.test`, to manually run FIPS test with specific agents. Unless the params.LABEL is also used to specify the labels instead of specific agent, that is the only reason I can think of.

Reply:
For FIPS testing, all tests have to run on the `ci.role.test.fips` machines. This is to prevent people from running fips testing on non-fips machines.